master/usage/configuration/mtu.md

```diff
@@ -28,8 +28,9 @@ packet.
 #### Default MTU sizes
 
 The default MTU for workload interfaces is 1500, this is to match the most
-common network MTU size.  The default MTU for the IP-in-IP tunnel device
-is 1440 to match the value needed in GCE.
+common network MTU size.  Moreover, the default MTU for the IP-in-IP tunnel
+device is also set to 1500. For GCE deployments, `FELIX_IPINIPMTU` should
```

Comment:
I think we want to leave this at 1440 - that's going to work in the most environments by default. For other environments, users can adjust as desired.